Create discord.md (#13)
polympics/wiki: content/discord.md

@title discord
@navbar

> # Polympics discord server
> 
> https://discord.gg/kJG7eM4hDG

Reviewer: ```suggestion
## [Polympics Discord Server](https://discord.gg/kJG7eM4hDG)

The official home of the event. Join to start chatting with fellow participants and team members!
```

Reviewer: Note that there should never be more that one single-`#` on a page, Google will rank pages lower for it.

> # Offciala Battle of Polytopia main discord server
> 
> https://discord.gg/polytopia

Reviewer: ```suggestion
## [Official Polytopia Discord Server](https://discord.gg/polytopia)

The official Discord server for The Battle of Polytopia.
```

## Note

> To register for the event please [login](login)

Reviewer: ```suggestion
To register for the event please [/login](login).
```
(Absolute URL is required.)
